packages/base/package.json

```diff
@@ -15,6 +15,9 @@
     "lit-html": "^1.0.0",
     "tslib": "^1.10.0"
   },
+  "devDependencies": {
+    "@material/ripple": "^3.0.0"
```

Comment:
This must be a dependency, because `MDCRippleFoundation` is the base class of an exported class. If a user imported it, and didn't happen to also depend on the ripple package, then TyepScript wouldn't be able to resolve that import.